examples/apps/aml/standalone-cipher/README.md

```diff
@@ -0,0 +1,10 @@
+## Python3 (at least 3.5.3) libraries needed to run the cipher
```

Comment:
Please add a license for documentation to the beginning of the file:
```
<!--
Licensed under Creative Commons Attribution 4.0 International License
https://creativecommons.org/licenses/by/4.0/
-->
```

Reply:
Add a sentence (or better yet, paragraph) to `examples/apps/aml/standalone-cipher/README.md` explaining what Standalone Cipher does.

Reply:
Would this application work on a docker build environment ? I don't see these pip packages installed in Dockerfile